Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [package]
+edition = "2018"
 name = "mizumochi"
 version = "0.1.0"
 authors = ["mopp <yuta_kobiyama@dwango.co.jp>"]
```

Comment:
This is just a comment.
I haven't had permission to maintain this PR.
So, it's time to change the authors.

Reply:
Thank you for pointing this out.
We will talk about this.